core/src/main/java/org/easymock/internal/MocksControl.java

```diff
@@ -373,6 +373,24 @@ public void asStub() {
         }
     }
 
+    public IExpectationSetters<Object> atLeast(int minimalNumberOfTimes) {
+        try {
+            state.times(new Range(minimalNumberOfTimes, Integer.MAX_VALUE));
+            return this;
+        } catch (RuntimeExceptionWrapper e) {
+            throw (RuntimeException) e.getRuntimeException().fillInStackTrace();
+        }
+    }
+
+    public IExpectationSetters<Object> atMost(int maximalNumberOfTimes) {
+        try {
+            state.times(new Range(1, maximalNumberOfTimes));
```

Comment:
Should the range start at zero or at one? From the method name i would have expected that never calling the mock is included in this expectation.